whylog_vim/consts.py

```diff
@@ -22,6 +22,7 @@ class Messages(object):
     LOGTYPE = 'This is the content and path of the line:'
     PRIMARY_KEY = 'This is the content and path of the line:'
     ADDED_EFFECT = '### WHYLOG ### You added line as effect. Select cause and press <F4>.'
+    WARNING_PREFIX = '! Warning ! %s'
```

Comment:
It is not warning prefix. It is warning template now, beacose has %s'. Change it.